app/controllers/tweet_controller.rb

```diff
@@ -0,0 +1,68 @@
+class TweetController < ApplicationController
+
+  get '/tweets' do
+    if logged_in?
+      @tweets = Tweet.all
+      @users = User.all
```

Comment:
I wanted all of the users available here, so that I can display a list of "Popular Users" on the all tweets page, in addition to displaying a list of tweets. So on the left 1/3 of the browser it shows a list of users. On the right 2/3 of the browser it list all tweets.